teams/rust-analyzer.toml

```diff
@@ -17,3 +29,6 @@ bors.rust-analyzer.review = true
 
 [[github]]
 orgs = ["rust-lang", "rust-analyzer"]
+
+[[zulip-groups]]
+name = "T-rust-analyzer"
```

Comment:
Once this is merged the `wg-rls2` zulip group can be removed which is not managed by the team repo